sphinx/builders/linkcheck.py

```diff
@@ -36,28 +38,25 @@ class HTMLParseError(Exception):
 from sphinx.util import encode_uri
 from sphinx.util.console import purple, red, darkgreen, darkgray, \
     darkred, turquoise
-from sphinx.util.pycompat import TextIOWrapper
```

Comment:
I think this file was the sole user of `sphinx.util.pycompat.TextIOWrapper`, so if it's no longer referenced we can deprecate/remove it.